src/pages/Signup.js

```diff
@@ -0,0 +1,49 @@
+import "../styles/auth.css";
+import {NavMenu} from "../components/NavMenu";
+import {Link} from "react-router-dom";
+const Signup = () => {
+    return (
+        <>
+        <NavMenu />
+        <section className="form-ctn br-md">
+        <form>
+            <h2 className="text-center mg-bottom-md">Signup</h2>
+            <div className="form-control">
+                <label htmlFor="first-name" className="fw-bold">First name</label>
+                <input type="text" name="first-name" placeholder="first name" />
+            </div>
+            <div className="form-control">
+                <label htmlFor="last-name" className="fw-bold">Last name</label>
+                <input type="text" name="last-name" placeholder="last name" />
+            </div>
+            <div className="form-control">
+                <label htmlFor="email" className="fw-bold">Email address</label>
+                <input type="text" name="email" placeholder="name@gmail.com" />
+            </div>
+            <div className="form-control">
+                <label htmlFor="password" className="fw-bold">Password</label>
+                <input type="password" name="password" placeholder="&lowast;&lowast;&lowast;&lowast;&lowast;&lowast;&lowast;&lowast;" />
+            </div>
+            <div className="form-control">
+                <label htmlFor="confirm-password" className="fw-bold">Confirm password</label>
+                <input type="password" name="confirm-password" placeholder="&lowast;&lowast;&lowast;&lowast;&lowast;&lowast;&lowast;&lowast;" />
+            </div>
+            <div className="form-control">
+                <input type="checkbox" name="checkbox" />
+                <label htmlFor="checkbox" className="fw-bold">I accept all Terms & Conditions</label>
```

Comment:
- change ```htmlFor``` attibute value from checkbox to its corresponding input element's ```id``` attribute value.

```suggestion
                <label htmlFor="t&c" className="fw-bold">I accept all Terms & Conditions</label>
```